src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/OrderItem.xml

```diff
@@ -36,6 +36,16 @@
             </itemOperation>
         </itemOperations>
 
+        <subresourceOperations>
+            <subresourceOperation name="adjustments_get_subresource">
```

Comment:
Would we also like to name it as in item or collection operation. I mean f.e. : 
```suggestion
            <subresourceOperation name="admin_adjustments_get_subresource">
```

Reply:
I cannot change that name, because this name is generated by the below configuration of subresource and thanks to that name, I can change the default path.
```
<property name="adjustments" readable="true">
    <subresource resourceClass="%sylius.model.adjustment.class%" />
</property>
```